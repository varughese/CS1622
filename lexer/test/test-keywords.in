else
if
int
return
void
while
ElSe
IF
els
els e
e lse
elsee
else9
eelse
9else
