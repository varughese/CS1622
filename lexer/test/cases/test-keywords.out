(1,KEY,"else")
(2,KEY,"if")
(3,KEY,"int")
(4,KEY,"return")
(5,KEY,"void")
(6,KEY,"while")
(7,ID,"ElSe")
(8,ID,"IF")
(9,ID,"els")
(10,ID,"els")
(10,ID,"e")
(11,ID,"e")
(11,ID,"lse")
(12,ID,"elsee")
(13,ID,"else9")
(14,ID,"eelse")
(15,NUM,"9")
(15,KEY,"else")
